### PR TITLE
[BD-6] Add tests in python 3.8

### PR DESCRIPTION
**Pull request by morenol**
Remove python 3.6 tests and add python 3.8 tests,
Upgrade dependencies
Fix quality issues

## Reviewers
- [ ] @ericfab170
- [ ] @awais786

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @morenol! I've created [OSPR-4492](https://openedx.atlassian.net/browse/OSPR-4492) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).

**Review by awais786 — APPROVED** [review]

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 language: python
 python:
-  - "3.5"
-  - "3.6"
+  - 3.5
+  - 3.8
 
 # command to install dependencies
 install:
```
Comment (ericfab179) [inline .travis.yml:7]: @morenol Do we want to move this requirements to  travis.in and travis.txt ?

**Review by ericfab179 — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (morenol) [reply .travis.yml:7]: > @morenol Do we want to move this requirements to  travis.in and travis.txt ?

Probably, that could be done in another PR

**Review by mraarif — APPROVED** [review]

**Comment by morenol** [comment]
Hi @coryleeio, this could be related to https://openedx.atlassian.net/browse/INFRA-98, the tests are already running in python 3

File: tubular/scripts/get_learners_to_retire.py [hunk 2]
```diff
@@ -55,7 +55,7 @@ def get_learners_to_retire(config_file,
     """
     if not config_file:
         click.echo('A config file is required.')
-        exit(-1)
+        sys.exit(-1)
```
Comment (awais786) [inline tubular/scripts/get_learners_to_retire.py:58]: @morenol is there any release note for this change ?

Comment (morenol) [reply tubular/scripts/get_learners_to_retire.py:58]: That was only to fix a pylint warning:
consider-using-sys-exit 

https://github.com/sushobhit27/pylint/commit/fa74fb744f1e71b65b162f817b6f02c4881e0acb

**Comment by openedx-webhooks** [comment]
@morenol 🎉 Your pull request was merged!

Please take a moment to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=edx/tubular&entry.867055334=https://github.com/edx/tubular/pull/418&entry.1484655318=https://github.com/morenol&entry.752974735=2020-05-07+18:47&entry.1917517419=2020-05-11+15:43&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**